src/usr/local/www/system_camanager.php

```diff
@@ -308,6 +317,7 @@
 
 		if (!$input_errors) {
 			write_config();
+			ca_trustedcerts_generate();
```

Comment:
Let me reduce it to basics you might get: To blindly assume the files are valid is more dangerous than to be safe and check them and verify or overwrite. The risks of assuming they are good outweigh the little benefit you get by skipping. 

Treating the bundle separately makes sense. It's from pkg, it isn't being changed based on user input like these CAs are. You're trying to lump things together that are clearly separate based on your own arbitrary criteria.

To put it another way, the bundle is to be considered static as a "Base system" file. The CA files are considered volatile because they are generated from user input.

I'm saying verify the CAs because we _can_ and it makes sense to do so here in this code.  It's safer. Touching the bundle does not since this code has nothing to do with the bundle.

The rewards of taking shortcuts are not great enough to counteract the risk.